# Instructions

I started to work on an html website to be hosted on my github repository based on a template on [HTML5 UP](http://html5up.net/). When you look through the repository you will see that within it (several files are obvi not made or written or edited by me, its placeholders).

I'd like your help with two tasks:

1. fixing the overall color pallete and aestethics of the website. Overall its just so dark and everything is hard to read and doesn't feel like good webdesign. I'm open to changing the background image on the homepage if needed for example.
2. Filling out the content for the rest of the webpage (of which I will describe below)

You can remove the What we Do section altogether I think. Im not sure what its really adding. I could always just put my CV on the website or just have a really nice contact page that has google scholar citations or some other cool stuff embedded in many.

___

## Research Projects

### Exploring the evolution and genomic architecture of non-canonical fungal specialized metabolism.

Passage to include with it:

I am interested in studying non-canonical fungal specialized metabolism which to me broadly encopasses two main ideas: (i) natural products whose genetic basis is poorly understood or unpredictable by conventional genome mining, and (ii) taxa who have been conventionally understudied or ignored by natural product researchers (e.g, Basideomycetes, lichens). My work on the fungal specialized metabolism is diverse and included are three relavent papers. I am always open to new collaborations on fungal natural product biosynthesis and evolution (see contact info page)! 

**List of relavent publications:**

Paper 1: 

path- paper_pdfs/gkad573.pdf

Title- Mining for a new class of fungal natural products: the evolution, diversity, and distribution of isocyanide synthase biosynthetic gene clusters

Scientific summary of work significance: 

My work provided the first large-scale characterization of isocyanide synthase (ICS) biosynthetic gene clusters (BGCs), a class of fungal natural products previously invisible to conventional genome mining. I developed a new computational pipeline that located over 3,800 ICS BGCs, establishing them as the fifth largest class of specialized metabolites in fungi. Our results, which included a full evolutionary analysis and the creation of a public web resource, provided a roadmap for future research. As a direct result of this work, the antiSMASH team incorporated detection of this new class into version 7.1 of their gold-standard genome mining tool.

Image- images/ICSOnTree_Final.png

Paper 2: (Make sure to mention it is a preprint and currently under review)

path- paper_pdfs/ssrn-5371150.pdf

Title- Reconstructing the evolution of a key fungal isocyanide megasynthase using genomes of lichenized fungi

Scientific summary of work significance: 

My work uncovered an abnormal "split" version of a known, antibiotic-producing isocyanide-NRPS mega-synthase (named CrmA) within lichen genomes. In this version, the domains that make up the mega-synthase are encoded by two separate genes (the ICS component is separate from the NRPS component).

We went on to show this split metabolic pathway is present in a wide range of fungi (not just lichens) and is functionally redundant with its fused counterpart. I traced the evolutionary origins of the NRPS-like domains to a duplication event of a canonical NRPS-like gene family found within the same fungal genomes. Our work demonstrates how exploring understudied fungal lineages can define new specialized metabolism lineages and reshape our understanding of biosynthetic pathways across the fungal kingdom.

Image- images/ProposedEvolutionaryNRPSSteps.png

Paper 3: (Make sure to mention it is a preprint and currently under review)

this one instead of having a path include a link to the following: https://www.proquest.com/openview/c23d63069f0d717ff649c259cbac4794/1?pq-origsite=gscholar&cbl=18750&diss=y

And mention: this paper is current unpublished but an early draft can be found within Grant's published Dissertation (Chapter 4)

Title- Equipped for success: Genomes and metabolomes of the European Amanita muscaria are conserved in its novel South African range

Scientific summary of work significance: 

This paper originated from a special, NSF-funded international research workshop in Pretoria, South Africa, in winter 2023. The *Amanita muscaria* species complex was introduced to South Africa in the context of forestry, but its origins, ecology, and recent evolution remained unstudied.

As a part of the workshop, our team sequenced the genomes of 24 Northern and Southern Hemisphere *A. muscaria*, built phylogenies, and reconstructed its South African history. My major contributions involved analyzing the specialized metabolomes of these mushrooms, using both genome mining and full chemical analysis with liquid chromatography-mass spectrometry (LC-MS/MS). I also led collaborations to test a panel of *A. muscaria* extracts against eight different organisms (including bacteria, nematodes, and insects).

The results of this paper are not printed but we hope it to be soon! 

Image- images/Figure4.png



___



### Aspergillus fumigatus Af293/CEA10 community sequencing initiative

I launched the *Aspergillus fumigatus* Community Sequencing Initiative in collaboration with Dr. Amelia Barber, Dr. Emile Gluck-Thaler, and Dr. Dante Calise. We founded this project on the principle that reliable, comparable data is crucial for scientific communities.

The model pathogen *A. fumigatus* (strains Af293 and CEA10) has been used by researchers for decades. As these strains were cultured independently in labs across the globe, they have inevitably accumulated unique, unstudied mutations. This "lab-to-lab" evolution makes it difficult to compare findings on physiology, pathogenicity, and genetics.

While community sequencing projects are common for other model organisms (like *Arabidopsis*, *S. cerevisiae*, and various bacteria), this effort is the first of its kind for a filamentous fungus. We believe *A. fumigatus* is the perfect model pathogen to pioneer this work. This is a global collaboration that we hope will provide a foundational genomic resource for the entire *Aspergillus* community for years to come.

To learn more about the project, see our FAQ, or get in touch, please visit the official initiative website.

https://asp-fumi-seq-project-2025.vercel.app/

___

### Understanding the genetic basis of these mixed microbial interactions and its potential use in environmental remediation of PFAS or the "forever chemicals".

To be continued!

The core aim of my work is to expand our understanding of the chemical interplay that occurs in mixed (fungal and bacterial) microbial communities. Toward this, I am developing two parallel approaches: running large-scale in silico protein docking experiments to identify candidate enzymes, and generating new, stable model fungal-bacterial communities whose collective metabolomes can be studied in a lab setting.

While these approaches may seem unconnected at first, I believe any viable microbial bioremediation strategy will ultimately involve the collective catabolic capabilities of many microbes working in concert. I hope to later build on the foundations of my model mixed microbial communities toward engineering consortia that can target and degrade specific chemicals, including PFAS.

